public/app/features/alerting/unified/components/silences/SilencesEditor.tsx

```diff
@@ -186,37 +182,19 @@ export const SilencesEditor = ({ silenceId, alertManagerSourceName }: Props) =>
     700,
     [clearErrors, duration, endsAt, prevDuration, setValue, startsAt]
   );
-
-  useDebounce(
-    () => {
-      // React-hook-form watch does not return referentialy equal values so this trick is needed
-      const newMatchers = matcherFields.filter((m) => m.name && m.value).map(matcherFieldToMatcher);
-      if (!isEqual(matchersForPreview, newMatchers)) {
-        setMatchersForPreview(newMatchers);
-      }
-    },
-    700,
-    [matcherFields]
-  );
-
   const userLogged = Boolean(config.bootData.user.isSignedIn && config.bootData.user.name);
 
-  if (getSilenceIsLoading) {
-    return <LoadingPlaceholder text="Loading existing silence information..." />;
-  }
-
-  const existingSilenceNotFound =
-    isFetchError(errorGettingExistingSilence) && errorGettingExistingSilence.status === 404;
-
-  if (existingSilenceNotFound) {
-    return <Alert title={`Existing silence "${silenceId}" not found`} severity="warning" />;
-  }
-
   return (
     <FormProvider {...formAPI}>
       <form onSubmit={handleSubmit(onSubmit)}>
-        <FieldSet>
-          <div className={cx(styles.flexRow, styles.silencePeriod)}>
+        <FieldSet className={styles.formContainer}>
+          {alertRule && (
+            <Field label="Alert rule">
+              <div>{alertRule.grafana_alert.title}</div>
+            </Field>
```

Comment:
I think this component could be more prominent that a regular form field. I find it difficult to spot the rule name. Maybe Deyan could help

Reply:
Dito, came to report the same thing but it looks like Konrad also noticed (or... not noticed 😆)